### ISSUE TITLE
Frequent and inconsistent "Unable to communicate with fork server (OOM?)"

### ISSUE THREAD

**Issue by reijin90** [Metalnem/sharpfuzz#11]
I'm seeing this error quite often with the Jil example:
```
[-] PROGRAM ABORT : Unable to communicate with fork server (OOM?)
         Location : run_target(), afl-fuzz.c:2417
```
I'm using AFL 2.56b, With dotnet 3.0. My csproj-file contains this:
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <OutputType>Exe</OutputType>
    <TargetFramework>netcoreapp3.0</TargetFramework>
  </PropertyGroup>

<ItemGroup>
  <Reference Include="Jil">
    <HintPath>Jil.dll</HintPath>
  </Reference>
</ItemGroup>
<ItemGroup>
  <PackageReference Include="SharpFuzz" Version="1.6.1" />
  <PackageReference Include="Sigil" Version="5.0.0" />
</ItemGroup>
</Project>
```

I've checked the `.cur_input` file, but when I add it to the testcases for the next run, it does not lead to a crash, so I'm guessing there is an other issue. In addition, when monitoring the memory usage there does not seem to be happening out of the ordinary: the process simply dies (disappears from `top`)

Can you help me investigate?

**Comment by Metalnem** [comment]
Can you try running afl-fuzz with the timeout parameter? For example: -t 5000 (this will set the execution timeout to five seconds). By default, afl-fuzz automatically calculates the timeout based on the previous runs, but that doesn't play really well with SharpFuzz.

**Comment by reijin90** [comment]
Thank you! that seems to have worked!

Did I miss that info in the docs? Or maybe we could add that somewhere?

**Comment by Metalnem** [comment]
You didn't miss it - I'll add it to the docs!

**Comment by Metalnem** [comment]
Fixed in 608faf96d95965aaccca2ed2e583bde165d83cea.
